frontend/src/generated/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
können wir drin lassen aber seems like this file is empty for now xD Interessanter generator... xD

Reply:
Ich lasses mal drin...wenns sich nicht iwann fuellt schmeis ichs wieder raus